lib/host/aws.js

```diff
@@ -83,32 +95,40 @@ export async function configure(baseUrl) {
     shutdown: {
       enabled: true,
       // Always wait 2 minutes minimum prior to shutting down this node.
-      minimumCycleSeconds: minutes(2),
+      minimumCycleSeconds: minutes(2)
     }
   };
 
-  log('metadata', config);
+  let userdata = await getJsonData(`${baseUrl}/user-data`);
+  let userSuppliedData = userdata.data;
+  let securityToken = userdata.securityToken;
+  let provisionerBaseUrl = userdata.provisionerBaseUrl;
```

Comment:
This is probably not more readable; i'm just showing off my skillz :)
```js
let {data, securityToken, provisionerBaseUrl} = userdata;
```
Also works....